b2sdk/bucket.py

```diff
@@ -171,6 +171,26 @@ def download_file_by_name(self, file_name, download_dest, progress_listener=None
             url, download_dest, progress_listener, range_
         )
 
+    def get_file_info_by_id(self, file_id):
+        """
+        Gets a file version's info by ID.
+
+        :param str file_name: the name of the file info to retrieve.
+        :rtype: generator[b2sdk.v1.FileVersionInfo]
+        """
+        response = self.api.session.get_file_info_by_id(file_id)
+        return FileVersionInfoFactory.from_api_response(response)
+
+    def get_file_info_by_name(self, file_name):
+        """
+        Gets a file version's info by its name.
+
+        :param str file_name: the name of the file info to retrieve.
+        :rtype: generator[b2sdk.v1.FileVersionInfo]
+        """
+        response = self.api.session.get_file_info_by_name(self.id_, file_name)
+        return FileVersionInfoFactory.from_api_response(response)
```

Comment:
My guess is actually this needs some coercion, so will test in eventually, but focusing on the unit tests first.